### PR TITLE
clarify how to load helpers inside cells

### PR DESCRIPTION
**Pull request by LordSimal**

### CODE REVIEW

**Review by Arhell — APPROVED** [review]

File: en/views/cells.rst [hunk 1]
```diff
@@ -276,3 +276,28 @@ This will allow us to define the option when creating the cell::
 Cell options are handy when you want data available as properties allowing you
 to override default values.
 
+Using Helpers inside a Cell
+===========================
+
+Since cells have their own View scope they don't share the helpers defined in
+`src/View/AppView.php`. You need to load those helpers again inside your cell::
```
Comment (othercorey) [inline en/views/cells.rst]: I think we use terms like "in your View" since it's not necessarily src/View/AppView.

Comment (ADmad) [inline en/views/cells.rst]: This is incorrect.  Cells have their own context and their own View instance but it's still an AppView instance and helpers loaded in AppView::initialize() would be available.

Comment (dereuromark) [reply en/views/cells.rst]: Defined in e.g. .. 
Might also work

Comment (LordSimal) [reply en/views/cells.rst]: I now have found my root problem. I thought I had loaded the plugins helper inside my AppView initialize method but I didn't. Thats why I was confused why the Helper wasn't available inside the cell.

I will adjust my statement.

File: en/views/cells.rst [hunk 2]
```diff
@@ -276,3 +276,30 @@ This will allow us to define the option when creating the cell::
...
+        protected $_validCellOptions = [];
+
+        public function initialize(): void {
+            $this->viewBuilder()->addHelper('MyCustomHelper');
+        }
+
+        public function display()
+        {
+            // Your code
+        }
```
Comment (saeideng) [inline en/views/cells.rst]: you can remove `display()`  function / `$_validCellOptions` and `use Cake\Datasource\Paginator;`  from sample

Comment (LordSimal) [reply en/views/cells.rst]: I thought I would stay with the same example used a little bit above but this seems fine as well

File: en/views/cells.rst [hunk 3]
```diff
@@ -276,3 +276,22 @@ This will allow us to define the option when creating the cell::
 Cell options are handy when you want data available as properties allowing you
 to override default values.
 
+Using Helpers inside a Cell
+===========================
+
+Cells have their own context and their own View instance but Helpers loaded inside your
+`AppView::initialize()` function are still loaded as usual.
```
Comment (markstory) [inline en/views/cells.rst]:
```suggestion
``AppView::initialize()`` function are still loaded as usual.
```

**Pull request merged**